ivr/src/main/resources/webapp/js/controllers.js

```diff
@@ -187,9 +202,16 @@
 
         $http.get('../ivr/ivr-configs')
             .success(function(response){
+                var i;
                 $scope.configs = response;
                 $scope.originalConfigs = angular.copy($scope.configs);
-                setAccordions($scope.configs);
+                setAccordions($scope.configs.configList);
+                for (i = 0 ; i < response.configs.length ; i = i + 1) {
```

Comment:
I still have TypeError: Cannot read property 'length' of undefined and Error: [ngRepeat:dupes] http://errors.angularjs.org/1.2.1/ngRepeat/dupes?p0=message%20in%20messages&p1=string%3ASettings%20updated%20successfully

in the browser console